src/store/awareness-store.ts

```diff
@@ -166,7 +167,16 @@ const reducer = ( state = DEFAULT_STATE, action: AwarenessAction ): AwarenessSto
 				}
 			} else {
 				const { userInfo } = action.payload.userState;
-				sendNotification( NotificationType.UserEntered, userInfo );
+				const currentMeUser = Array.from( state.userMap.values() ).find(
+					user => user.userInfo.isMe && user.userInfo.isConnected
+				);
+
+				sendNotification(
```

Comment:
I had originally tried to keep the checks in here, but we want to keep all the notification sending logic in one place which is why it's been moved inside here. 

The function is a bit messy, so next time we add anything we may want to create separate functions perhaps for each notification type.